src/Microsoft.AspNetCore.SignalR.Common/Internal/Protocol/JsonHubProtocol.cs

```diff
@@ -111,6 +111,8 @@ private HubMessage ParseMessage(TextReader textReader, IInvocationBinder binder)
                 using (var reader = new JsonTextReader(textReader))
```

Comment:
Use new `JsonUtils.CreateJsonTextReader(textReader)` method to create JsonReader